src/Sql/Sql/Database/Cmdlet/NewAzureSqlDatabase.cs

```diff
@@ -205,7 +205,7 @@ public class NewAzureSqlDatabase : AzureSqlDatabaseCmdletBase<AzureSqlDatabaseCr
         /// </summary>
         [Parameter(Mandatory = false,
             HelpMessage = "The Backup storage redundancy used to store backups for the SQL Database. Options are: Local, Zone and Geo.")]
-        [ValidateSet("Local", "Zone", "Geo")]
+        [ValidateSet("Local", "Zone", "Geo", IgnoreCase = false)]
```

Comment:
this looks like a breaking change for me, do you want to release this in October major release?

Reply:
I'd like this to go as soon as possible; this is added to a parameter, BackupStorageRedundancy, that was only introduced in the last milestone  (introduced in S175 (2020-09-22) Ignite).  @dingmeng-xue and @msJinLei have reviewed my previous changes that this belongs to. 

I will add change log too.

Reply:
merged by mistake, the change log should under upcoming release, I'll make another PR to fix it